msal/src/main/java/com/microsoft/identity/client/AuthenticationActivity.java

```diff
@@ -24,22 +24,18 @@
 package com.microsoft.identity.client;
 
 import android.app.Activity;
+import android.content.ComponentName;
```

Comment:
AuthenticationActivity is deprecated. Here, I just revert the change according to the master branch.